tasks/elasticsearch-templates.yml

```diff
@@ -29,5 +29,7 @@
   register: resultstemplate
 
 - name: Install template(s)
-  command: "curl -sL -XPUT http://localhost:{{http_port}}/_template/{{item}} -d @/etc/elasticsearch/templates/{{item}}.json"
+  command: "curl -sL -XPUT --output /dev/null --write-out %{http_code} http://localhost:{{http_port}}/_template/{{item}} -d @/etc/elasticsearch/templates/{{item}}.json"
```

Comment:
I think it would better to the [uri module](http://docs.ansible.com/ansible/uri_module.html) to interact with the API.